src/main/scala/uk/gov/hmrc/wco/dec/Declaration.scala

```diff
@@ -66,25 +68,21 @@ trait JacksonMapper {
 @JsonIgnoreProperties(Array("_xml", "_schema", "_props"))
 @JacksonXmlRootElement(namespace = NS.dms, localName = "MetaData")
 case class MetaData(@JacksonXmlProperty(localName = "WCODataModelVersionCode", namespace = NS.dms)
-                    wcoDataModelVersionCode: Option[String] = None, // max 6 chars
-
+                    wcoDataModelVersionCode: Option[String] = None,
                     @JacksonXmlProperty(localName = "WCOTypeName", namespace = NS.dms)
-                    wcoTypeName: Option[String] = None, // no constraint
-
+                    wcoTypeName: Option[String] = None,
                     @JacksonXmlProperty(localName = "ResponsibleCountryCode", namespace = NS.dms)
-                    responsibleCountryCode: Option[String] = None, // max 2 chars - ISO 3166-1 alpha2 code
-
+                    responsibleCountryCode: Option[String] = None,
                     @JacksonXmlProperty(localName = "ResponsibleAgencyName", namespace = NS.dms)
-                    responsibleAgencyName: Option[String] = None, // max 70 chars
-
+                    responsibleAgencyName: Option[String] = None,
                     @JacksonXmlProperty(localName = "AgencyAssignedCustomizationCode", namespace = NS.dms)
-                    agencyAssignedCustomizationCode: Option[String] = None, // max 6 chars
-
+                    agencyAssignedCustomizationCode: Option[String] = None,
                     @JacksonXmlProperty(localName = "AgencyAssignedCustomizationVersionCode", namespace = NS.dms)
-                    agencyAssignedCustomizationVersionCode: Option[String] = None, // max 3 chars
-
+                    agencyAssignedCustomizationVersionCode: Option[String] = None,
                     @JacksonXmlProperty(localName = "Declaration", namespace = NS.dec)
-                    declaration: Declaration = Declaration()) extends JacksonMapper {
+                    declaration: Option[Declaration] = None,
+                    @JacksonXmlProperty(localName = "Response", namespace = NS.res)
+                    response: Option[Seq[Response]] = None) extends JacksonMapper {
```

Comment:
Why do we need `Option[Seq[Response]]` rather than just `Seq[Response]` which defaults to `Seq.empty`?

Reply:
@raghu1978

Reply:
@PatrykRudnicki I think we have missed this while we are trying to sort out response, please change this with Chris suggestion

Reply:
@raghu1978 Done :)